README.md

```diff
@@ -1,30 +1,34 @@
 linter-flake8
 =============
 
-linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8 provider for [Linter](https://github.com/AtomLinter/Linter).
+linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8) provider for the Atom [Linter](https://github.com/atom-community/linter) package
 
 ![img](https://cloud.githubusercontent.com/assets/4278113/8768482/52f975c6-2e3f-11e5-87e4-27c8359fd36c.gif)
 
 ### Requirements
-Linter package must be installed in order to use this plugin. If it's not
-installed, please follow the instructions [here](https://github.com/AtomLinter/Linter).
+The `Linter` package must be installed in order to use this package. If it's not installed, please follow [Linter's installation instructions](https://github.com/atom-community/linter#how-to--installation).
 
 ### Installation
-To use a plugin, you must ensure that `flake8` is installed on your
-system.
-
-Install [flake8](https://pypi.python.org/pypi/flake8) by typing the following
-in a terminal:
+First, ensure that [flake8](https://pypi.python.org/pypi/flake8) is installed on your system. If not, install it by typing the following in a terminal:
    ```
    pip install flake8
    ```
 
-Install plugin by typing:
+Install package by typing:
    ```
-   $ apm install linter-flake8
+   apm install linter-flake8
    ```
 
-##### Bult-in docstrings check (Optional)
+#### Virtualenv
+
+To ensure `flake8` targets the proper Python version for each project you work on, it is highly recommended to install flake8 in each project [virtualenv](https://virtualenv.pypa.io/en/latest/):
+
+* If you are okay with installing flake8 in each virtualenv, just do that.
+* If you'd rather use system-wide flake8 versions (e.g. `flake8` and `flake8-python2`), create a `flake8` symlink in each virtualenv pointing to your desired flake8 version. For example: `ln -s /usr/bin/flake8-python2 ~/.virtualenvs/random_python2_project/bin/flake8`
+
+Then [activate your virtualenv](https://virtualenv.pypa.io/en/latest/userguide.html#activate-script) from the command line before starting Atom, and you're good to go!
+
+##### Built-in docstrings check (Optional)
 To include built-in docstrings (pep257) support you will also need to install:
    ```
    pip install flake8-docstrings
```

Comment:
We might also mention [`flake8-pep257`](https://pypi.python.org/pypi/flake8-pep257) which does the same thing.

`flake8-docstrings` actually doesn't work with this package right now (at least on Windows), although the author is [working on it](https://gitlab.com/pycqa/flake8-docstrings/issues/2).